src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -647,7 +647,7 @@ private static bool IsTypeLessVisibleThan(TypeSymbol type, Symbol sym, ref Compo
                 typePredicate: predicate,
                 arg,
                 canDigThroughNullable,
-                visitCustomModifiers);
+                visitCustomModifiers: visitCustomModifiers);
```

Comment:
Do we have analyzers for this kind of mistake? It was passing to the wrong parameter.